examples/benchmark.py

```diff
@@ -102,8 +127,28 @@
     for idx in range(len(performance)):
         row = "%d x %d" % (resolutions[idx], resolutions[idx])
         for key, value in performance[idx].items():
-            row += "\t%-8.1f" % value
+            row += "\t%-8.1f" % value.get("fps")
         print(row)
     print(
         " =============================================================================="
     )
+
+    # also print the average time per simulation step (including object perturbations)
+    if args.enable_physics:
+        print(
+            " ================ Performance (step time: seconds) NPROC={} ===================================".format(
```

Comment:
Thanks for showing also step time and not just FPS :+1: A minor thing: could the frame time be in milliseconds instead of seconds? There's a lot of leading zeros, which makes a difference between 0.00012 and 0.0012 look less significant than it would with 0.12 and 1.2 :)